packages/lesswrong/lib/collections/comments/fragments.ts

```diff
@@ -136,6 +139,8 @@ registerFragment(`
   }
 `);
 
+// TODO: This is now the same as CommentWithRepliesFragment, now that said
+// fragment gets the tag field
 registerFragment(`
   fragment StickySubforumCommentFragment on Comment {
     ...CommentWithRepliesFragment
```

Comment:
@jpaddison3 Is this a TODO you intended to address in your PR?

Reply:
Nope

Reply:
(I didn't introduce the reproduction)